tests/epyccel/modules/strings_module.py

```diff
@@ -47,3 +49,12 @@ def concatenate_expr():
     t = "world"
     v = s + "_" + t
     return v
+
+def string_function_call():
+    s = 'hello'
+    t = str(s)
+    return t
+
+def string_function_call_on_literal():
+    t = str('hello')
+    return t
```

Comment:
What happens if we return the output of `str` directly,  without the temporary variable `t`?

Reply:
I have added a test: [6a0d593](https://github.com/pyccel/pyccel/pull/2200/commits/6a0d5933155563c349ba2a59f750e04caf014791)

It works as expected. In C:
```c
return cstr_lit("hello");
```
In Fortran:
```fortran
result_0001 = 'hello'
```